lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -344,6 +349,22 @@ dictionary CheckMessageResponse {
     boolean is_valid;
 };
 
+dictionary PrepareLnUrlPayRequest {
+    LnUrlPayRequestData data;
+    u64 amount_msat;
+    string? comment = null;
+    boolean? validate_success_action_url = null;
+};
+
+dictionary PrepareLnUrlPayResponse {
+    PrepareSendResponse prepare_send_response;
```

Comment:
IMO it is better to have the destination and fee_sat directly under the PrepareLnUrlPayResponse and not use the PrepareSendResponse for that. It reduces decoupling between the two methods.

Reply:
Agreed. Done [27313d5](https://github.com/breez/breez-sdk-liquid/pull/524/commits/27313d5616373424875cfd58396aab99271b8957)